content/batch/workflow-simple.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Simple Batch Workflow"
+date: 2018-11-18T00:00:00-05:00
+weight: 60
+draft: false
+---
+
+### Simple Batch Workflow
+
+Let's deploy the `whalesay` example from before using Argo.
```

Comment:
Save the below manifest as 'workflow-whalesay.yaml' using your favorite editor and let's deploy...

Reply:
Updated in future commit.